dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -94,10 +94,10 @@ export const YoutubeBlockComponent = ({
 	const [consentState, setConsentState] = useState<ConsentState | undefined>(
 		undefined,
 	);
-
 	const abTests = useAB();
+	const abTestsApi = abTests?.api;
```

Comment:
This fixes a ts error and brings this code into line with other uses of `useAB()`